src/main/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPlugin.groovy

```diff
@@ -43,7 +43,9 @@ class DockerRemoteApiPlugin implements Plugin<Project> {
         // if no repositories were defined fallback to buildscript
         // repositories to resolve dependencies as a last resort
         if (project.repositories.size() == 0) {
```

Comment:
@cdancy I agree with @stempler that the repository count check should be inside `afterEvaluate` closure.